runtime/near-vm-runner/Cargo.toml

```diff
@@ -22,6 +22,7 @@ wasmer-types = { version = "1.0.2", optional = true }
 wasmer-compiler-singlepass = { version = "1.0.2", optional = true }
 wasmer-compiler-cranelift = { version = "1.0.2", optional = true }
 wasmer-engine-native = { version = "1.0.2", optional = true }
+wasmer-vm = { version = "1.0.2" }
```

Comment:
should probably be `optional = true` for now, like the rest of wasmer 1 crates?

Reply:
Would love to, but unfortunately cannot get it compiled in such a way.